docs/content/latest/api/ysql/datatypes/type_datetime/_index.md

```diff
@@ -42,14 +42,58 @@ The <a href="https://www.postgresql.org/docs/11/datatype-datetime.html#DATATYPE-
 {{< /note >}}
 
 Modern applications almost always are designed for global deployment. This means that they must accommodate timezones—and that it will be the norm therefore to use the _timestamptz_ data type and not _date_, plain _time_, or plain _timestamp_. Application code will therefore need to be aware of, and to set, the timezone. It's not uncommon to expose the ability to set the timezone to the user so that _date-time_ moments can be shown differently according to the user's present purpose.
 
+## Special date-time manifest constants
+
+PostgreSQL, and therefore YSQL, support the use of several special manifest _text_ constants when they are typecast to specified _date-time_ data types, thus:
+
+| constant    | valid with                         |
+| ----------- | ---------------------------------- |
+| 'epoch'     | date, plain timestamp              |
+| 'infinity'  | date, plain timestamp, timestamptz |
+| '-infinity' | date, plain timestamp              |
+| 'now'       | date, plain time, plain timestamp  |
+| 'today'     | date, plain timestamp              |
+| 'tomorrow'  | date, plain timestamp              |
+| 'yesterday' | date, plain timestamp              |
+| 'allballs'  | plain time                         |
+
+Their meanings are give in section [8.5.1.4. Special Values](https://www.postgresql.org/docs/11/datatype-datetime.html#DATATYPE-DATETIME-SPECIAL-VALUES) in the PostgreSQL documentation.
+
+{{< tip title="Avoid using all of these special constants except for _'infinity'_ and _'-infinity'_." >}}
+
+The implementation of the function [random_test_report_for_modeled_age()](./functions/miscellaneous/function-age/#function-random-test-report-for-modeled-age) shows a common locution where _'infinity'_ and _'-infinity'_ are used to initialize maximum and minimum values for _timestamp_ values that are updated as new _timestamp_ values arise during a loop's execution.
+
+The constants _'infinity'_ and _'-infinity'_ can be also used to define _range_ values that are unbounded at one end. But this effect can be achieved also, and arguably with more clarity. simply by omitting the value at the end of the range that you want to be unbounded.
+
+The remaining special constants have different kinds of non-obvious results. See the recommendation [Don't use the special manifest constant 'now'.](./functions/#avoid-constant-now) on the 'General-purpose date and time functions' page. The constants _'today'_, _'tomorrow'_. and _'yesterday'_ all bring analogous risks to those brought by _'now'_. And the intended effects of _'epoch'_ and _'allballs'_ are brought with optimal clarity for the reader by typecasting an appropriately spelled literal value to the required data type, whatever it might be.
+
+Yugabyte recommends that you avoid using all of the special manifest _text_ _date-time_ constants except for for _'infinity'_ and _'-infinity'_.
```

Comment:
```suggestion
Avoid using all of the special manifest _text_ _date-time_ constants except for for _'infinity'_ and _'-infinity'_.
```

Reply:
No, I'm keeping _"Yugabyte recommends"_. It's a normal doc style. The Oracle Database documentation uses it a lot. I just a global search within the entire _ysql_ tree, I counted _47_ occurrences.